docs/minio-client-complete-guide.md

```diff
@@ -3,41 +3,34 @@
 Minio Client (mc) provides a modern alternative to UNIX commands like ls, cat, cp, mirror, diff etc. It supports filesystems and Amazon S3 compatible cloud storage service (AWS Signature v2 and v4).
 
 ```sh
-
-ls            List files and folders.
-mb            Make a bucket or folder.
-cat           Display contents of a file.
-pipe          Write contents of stdin to target. When no target is specified, it writes to stdout.
-share         Generate URL for sharing.
-cp            Copy one or more objects to a target.
-mirror        Mirror folders recursively from a single source to single destination.
-diff          Compute differences between two folders.
-rm            Remove file or bucket [WARNING: Use with care].
-events        Manage bucket notification.
-watch         Watch for events on object storage and filesystem.
-policy	      Set public policy on bucket or prefix.
-session       Manage saved sessions of cp and mirror operations.
-config        Manage configuration file.
-update        Check for a new software update.
-version       Print version.
-
+ls       List files and folders.
+mb       Make a bucket or a folder.
+cat      Display file and object contents.
+pipe     Redirect STDIN to an object or file or STDOUT.
+share    Generate URL for sharing.
+cp       Copy files and objects.
+mirror   Mirror buckets and folders.
+find     Finds files which match the given set of parameters.
+diff     List objects with size difference or missing between two folders or buckets.
+rm       Remove files and objects.
+events   Manage object notifications.
+watch    Watch for files and objects events.
```

Comment:
`Watch for files and objects events.`
 ==>
`Watch for file and object events.`